index.ts

```diff
@@ -1,16 +1,24 @@
 import Web3 = require('web3')
-import * as transport from './lib/transport'
-import { default as Storage, engine, build, channels as storageChannels } from './lib/storage'
+import { default as Storage, build } from './lib/storage'
 import Engine from './lib/engines/engine'
 import * as channel from './lib/channel'
-import { default as Sender } from './lib/sender'
-import { ChannelContract } from './lib/channel'
+import { ChannelContract, id } from './lib/channel'
 import { PaymentChannel } from './lib/paymentChannel'
 import * as BigNumber from 'bignumber.js'
 import Payment from './lib/Payment'
-import * as receiver from './lib/receiver'
 import { TransactionResult } from 'truffle-contract'
-import serviceRegistry, { Container } from './lib/container'
+import mainRegistry, { Container, Registry } from './lib/container'
+import ChannelManager from './lib/channel_manager'
+import ChannelsDatabase from './lib/storages/channels_database'
+import Client, {
+  AcceptPaymentRequest, AcceptPaymentResponse, AcceptTokenRequest,
+  AcceptTokenResponse
+} from './lib/client'
+import { PaymentRequired } from './lib/transport'
+
+require('./lib/channel_manager')
```

Comment:
@mslipper What is that?

Reply:
Since we're only importing an interface, the emitted JS code never requires the channel manager and so its services never get registered.

Reply:
That does not look right for me. Could we change that to a something more explicit?